docs/contributing/contributing-to-airbyte/updating-documentation/source-documentation-template.md

```diff
@@ -1,25 +1,29 @@
-# Source Documentation Template
+---
```

Comment:
John changed the capitalization of headings, make sure you adapt to it (template and all the integration docs)